### DISCUSSION TITLE
localproxy with `tcp_proxy` plugin for processing logging of Trannergy inverters

### DISCUSSION THREAD

**Discussion by hansij66** [jbouwh/omnikdatalogger#69] [Ideas]
Jan,

I am having a Trannergy inverter; afaik, same family as Omnik. For the Trannergy inverter, I don't need the localproxy work around. You can set a "remote server" which points to the server which runs the data collection script. See attached screenshot.

![image](https://user-images.githubusercontent.com/21691535/153760090-da99f9fb-2ed0-4743-ba4b-e1f412cb7552.png)

I was looking for a means to collect data at a higher frequency....I run into your tcpclient, which works for my Trannergy inverter; thanks!

Cheer
Hans

**Comment by jbouwh** [comment]
I'll see if I can post your info on my website to help other users. Thnx for sharing.

**Reply by jbouwh** [reply]
Changed the issue to a discussion. I think you can use the `tcp_proxy` plugin of `local_proxy` to be able to proces the inverter updates directly. You do not need `omnikdataloggerproxy` or NAT rules in this case.
